resources/user-agents/browsers/chrome/headless-chrome.json

```diff
@@ -21,7 +21,7 @@
           "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) HeadlessChrome Safari/*",
           "device": "Linux Desktop",
           "platforms": [
-            "Linux_64", "Linux_32_x86", "Linux_32", "Linux"
```

Comment:
Is it possible to remove these two platforms completely (also from the platforms file)?

Reply:
I'm not sure yet, I only looked for cases where they were used on the same pattern as the "Linux" platform.

I can take a look though.

Reply:
Looks like they can be removed (and another platform that just extended "Linux" but didn't change any properties that wasn't actually used anywhere).  There were a couple of standard specific tests that were testing for "unknown" platform when I changed `Linux_32` to `Linux` in a couple of cases, but I've removed those tests, and changed the `standard` prop on the tests they were derived from to `true`.

Update coming.